polyfills/Array/from/config.json

```diff
@@ -7,11 +7,12 @@
 	"browsers": {
 		"chrome": "<45",
 		"firefox": "4 - 31",
-		"ie": "6 - *",
+		"ie": "6 - 11",
 		"ie_mob": "10 - *",
 		"opera": "*",
-		"safari": "7 - *",
-		"firefox_mob": "4 - *"
+		"safari": "<9",
+		"firefox_mob": "4 - *",
+		"ios_saf": "3 - 8"
```

Comment:
According to [Kangax's compatibility table](https://kangax.github.io/compat-table/es6/#test-Array_static_methods)iOS 9 has partial support for Array.from - it is missing:
Array[Symbol.species]
Array.from, generator instances
Array.from map function, generator instances

Our polyfill does not support these features either IIRC.